examples/SharedMemory/TinyRendererVisualShapeConverter.cpp

```diff
@@ -244,14 +292,6 @@ void convertURDFToVisualShape(const UrdfVisual* visual, const char* urdfPathPref
 			visualShapeOut.m_dimensions[1] = visual->m_geometry.m_meshScale[1];
 			visualShapeOut.m_dimensions[2] = visual->m_geometry.m_meshScale[2];
 
-			visualShapeOut.m_localVisualFrame[0] = visual->m_linkLocalFrame.getOrigin()[0];
```

Comment:
You mean, from URDF_GEOM_MESH. Because it overrides defaults set by caller, defaults are the same.

(Of course please check again, I can very well make a mistake)